Cargo.toml

```diff
@@ -27,7 +27,7 @@ unwrap = "~1.1.0"
 
 [dev-dependencies]
 docopt = "~0.6.86"
-libc = "~0.2.16"
+libc = "~0.2.17"
```

Comment:
I just did the changes manually (I think cargo update would only touch the lock file anyway, but there is a tool for updating the toml file which some of the other guys use I think), but we don't really have an agreed-upon approach.

I generally try and keep Cargo.toml up to date whenever I remember and have been just adding any changes as part of unrelated commits.